Deploy-CAM.ps1

```diff
@@ -1499,11 +1497,12 @@ function New-ConnectionServiceDeployment() {
                 -Verbose
         }
         else {
-            for($idx = 10;$idx -gt 0;$idx--)
+            $maxRetries = 30
```

Comment:
I hit this a lot yesterday, sometimes requiring 15 retries. I bumped the threshold to 30.